Portal/src/Datahub.Portal/Components/Buttons/DHButton.razor

```diff
@@ -21,7 +21,9 @@
            Tag="@Tag"
            UserAttributes="@UserAttributes"
            OnClick="@OnClick">
-    @ChildContent
+    <MudText>
+        @ChildContent
+    </MudText>
```

Comment:
What if I want a button that's not MudText? Like an icon or image button?

Reply:
Do you have an example where we use `DHButton` this way? Sorry, just trying to wrap my head around the issue. I think an Icon would be using `MudIconButton` component. The `MudText` is to allow normal casing in standard buttons and is how MudBlazor documentation does that.

@davidreneuw Do you have any input on this?

Reply:
> @davidreneuw Do you have any input on this?
Im not sure why this is necessary. If someone who uses DHButton wants the childcontent to be in a mudtext element, then they can add it themselves (i.e. <DHButton><MudText/></DHButton>). By making this the default, we lost some options

Reply:
I've reverted DHButton. I've kept the topbar/footer navigation buttons with standard casing by adding the MudText to each.

![image](https://github.com/user-attachments/assets/d992b889-196c-4d96-b927-f076717645e7)

Reply:
If we want the top buttons to look like links would it make more sense to use MudLinks instead?

Reply:
I've reverted the underlining, thought the mockup showed it but was wrong!

Reply:
Ah gotcha